docker-app/qfieldcloud/core/admin.py

```diff
@@ -1037,7 +1038,71 @@ def has_change_permission(self, request, obj):
         return False
 
 
+class ActiveUsersFilter(admin.SimpleListFilter):
+    title = _("active")
+    parameter_name = "active_users"
+
+    def lookups(self, request: Any, model_admin: Any) -> List[Tuple[Any, str]]:
+        return [
+            ("only_with_active_users", _("Yes")),
+            ("only_without_active_users", _("No")),
+        ]
+
+    def queryset(self, request: HttpRequest, qs: QuerySet) -> QuerySet:
+        if self.value() == "only_active_users":
+            return qs.filter(
+                useraccount__current_subscription_vw__isnull=False
+            ).order_by("+useraccount__current_subscription_vw")
```

Comment:
does that work

Reply:
Removed the "+". Good catch.

Reply:
Do we have any admin tests yet? can you try to somehow have this and other new admin features test covered? maybe we can just somewhere load admin pages list and navigate to one detail view ...

Reply:
Hum, this PR is already 200 lines long and the business logic of the new feature (= counting jobs per organization and users) is already tested in a test that is part of this PR. There are two other changes which are basically pure Django-wiring:
- using the modelForm field as a source of data for the widget
- adding a new filter 

Wouldn't adding 2 extra tests amount to testing Django?

Reply:
@faebebin is that OK or not yet? If not yet, maybe we can talk about this tomorrow, because I'd be interested if you have more comments / suggestions on other parts of this PR.

Reply:
will get back to it in afternoon

Reply:
> Do we have any admin tests yet? can you try to somehow have this and other new admin features test covered? maybe we can just somewhere load admin pages list and navigate to one detail view ...

There is a test checking if admin pages are loading. See here https://github.com/opengisch/qfieldcloud/blob/master/docker-app/qfieldcloud/core/tests/test_admin.py#L62

Reply:
I would suggest focus on the core of the task first. You can branch off to continue later on filter or sorting and maybe even add a follwoup ticket? And then let filter / sort aside for now ?